scanners/zap-advanced/Makefile

```diff
@@ -30,6 +30,7 @@ kind-import-parser:
 	cd $(SCANNERS_DIR)/zap/ && $(MAKE) -s kind-import-parser
 
 deploy-with-scanner:
+	rm -rf venv # delete venv. Otherwise it would be deployed with zap
```

Comment:
This means that one need to reininstall the whole venv all the time which is time consuming. Is it possible to ignore the directory? I wonder if there is something like `.dockerignore`, maybe `.helmignore`?

Unless you should use a variable for the dir `venv` because as mentioned above the directory name can be chosen freely.